src/api/about.coffee

```diff
@@ -0,0 +1,21 @@
+currentCoreVersion = require('../../package.json').version
+aboutInfoObject = require '../../config/about.json'
+
+logger = require 'winston'
+utils = require '../utils'
+authorisation = require './authorisation'
+
+exports.getAboutInformation = () ->
+  
+  if not authorisation.inGroup 'admin', this.authenticated
+    return utils.logAndSetResponse this, 403, "User #{this.authenticated.email} is not an admin, API access to addClient denied.", 'info'
```

Comment:
I think this check can be removed as all logged in users should be able to see what version is running